roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -107,16 +112,16 @@ def is_image_local(self, image, task_vars):
 
         return bool(result.get("images", []))
 
-    def known_docker_registries(self, task_vars):
-        result = self.module_executor("docker_info", {}, task_vars)
+    @staticmethod
+    def known_docker_registries(task_vars):
+        docker_facts = get_var(task_vars, "openshift", "docker")
+        regs = set(docker_facts["additional_registries"])
 
-        if result.get("failed", False):
-            return []
+        deployment_type = get_var(task_vars, "openshift_deployment_type")
+        if deployment_type == "origin" and "docker.io" not in regs:
```

Comment:
Don't need to check `"docker.io" not in regs`, update is idempotent

Can we do the same for enterprise? https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.ose.example#L74